packages/curriculum-compiler-json/__tests__/fixtures/glossary/sample/ast.json

```
@@ -0,0 +1,246 @@
+{
+  "type": "root",
+  "children": [
+    {
+      "type": "heading",
+      "depth": 1,
+      "children": [
+        {
+          "type": "text",
+          "value": "Markdown Formatting"
+        }
+      ]
+    },
+    {
+      "type": "paragraph",
+      "children": [
+        {
+          "type": "text",
+          "value": "Just as we use punctuation to improve readability, we can use markdown to format text in our comment section."
+        }
+      ]
+    },
+    {
+      "type": "paragraph",
+      "children": [
+        {
+          "type": "text",
+          "value": "Before you leave your first submission, it might help to brush up your markdown."
+        }
+      ]
+    },
+    {
+      "type": "paragraph",
+      "children": [
+        {
+          "type": "text",
+          "value": "When submitting an answer in the comment section, you'll want to use something like this:"
+        }
+      ]
+    },
+    {
+      "type": "code",
+      "lang": "py",
+      "meta": null,
+      "value": "print(\"So this is markdown!\");\nthis should "
```

should this be like this?
```suggestion
      "value": "print(\"So this is markdown!\");"
```

---

doesn't matter, this is a test; any md is good